### DISCUSSION TITLE
folder structure: year/month/photo

### DISCUSSION THREAD

**Discussion by Gumeljohn** [immich-app/immich#14550] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello Immich Team!

Thank you for your great work on the project. I have a problem. If I upload photos to Immich, they are in completely different locations. Suggestion for improving Immich functionality. It would be very useful if Immich could automatically organize photos on disk by year and month, based on EXIF data. This would make it easier to navigate through files when accessing them directly through the file system.

**Advantages**:
1. Convenience: It is easier to find photos when they are organized by date.
2. Organization: Keeps the file system in order.
3. Compatibility: Simplifies backup and synchronization with other devices.

**Suggestion**:
Add an option in Immich settings to automatically organize files on disk by year and month.

Thanks for your attention to my suggestion!

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
https://immich.app/docs/administration/storage-template
